### PR TITLE
Fail if neither folder or code is expected in DatadirTestsFromDirectoryProvider

### LINKED ISSUE #19: Either "expected folder" or "expected code" should be required in DatadirTestsFromDirectoryProvider
In this commit https://github.com/keboola/processor-remove-empty-files-and-folders/pull/3/commits/4100e401b7e2fd6bde07ace2703d4a9315ed81e0 there is not `expected` folder for `tests/functional/blank-file-with-empty-lines/` and the test is passing

### PR DESCRIPTION
**Pull request by tomasfejfar**
Fixes #19

### CODE REVIEW

File: src/DatadirTestsFromDirectoryProvider.php [hunk 1]
```diff
@@ -92,6 +93,13 @@ protected function processOneTest(SplFileInfo $testSuite): void
             $expectedOutputDirectory = $outTemplateDir;
         }
 
+        if ($expectedOutputDirectory === null && $expectedReturnCode === null) {
+            throw new LogicException(sprintf(
+                '%s: Either "expected/out/data" folder or "expected-code" file must exist',
```
Comment (odinuv) [inline src/DatadirTestsFromDirectoryProvider.php]: At least one of ... must exist ?

**Review by odinuv — APPROVED** [review]

File: src/DatadirTestsFromDirectoryProvider.php (hunk 1 above)
Comment (tomasfejfar) [reply src/DatadirTestsFromDirectoryProvider.php]: Jo, máš pravdu já to mám jako XOR a ono je to OR.

Comment (tomasfejfar) [reply src/DatadirTestsFromDirectoryProvider.php]: Tyjo, ale zní to úplně debilně imho :( `At least one of "expected/out/data" folder or "expected-code" file must exist` @pivnicek ?

Comment (pivnicek) [reply src/DatadirTestsFromDirectoryProvider.php]: hmmm, how about. `Neither the "expected/out/data" folder nor the "expected-code" file exist` which is frankly an unacceptable situation :-)

Comment (pivnicek) [reply src/DatadirTestsFromDirectoryProvider.php]: but I think `at least one of ... must exist` is also fine

**Pull request merged**